### ISSUE TITLE
FSR3.1 Upscaler cant be applied

### ISSUE THREAD

**Issue by WATTASIGMA** [cdozdil/OptiScaler#39]
Hello,

maybe someone can help me out or enlighten me on this problem I have. First off this repo (mod) is awesome and thank you everyone who was/is involved into this project. So my problem is in the .ini I set the upscaler to "fsr31" but ingame if I open the OptiScaler menu it shows me I'm using FSR 2.1.2 and if I apply FSR 3.1 it reverts it back to FSR 2.1.2 but FSR 2.2.1 works fine. So the mod is ignoring the .ini and uses FSR 2.1.2. I also confirmed this by checking the visual quality in case its just a menu bug or something and sadly it uses FSR 2.1.2. Maybe my GPU doesnt support it but other games which supports FSR 3 natively it works perfectly and even FSR Frame-Gen is also working. My GPU is a Vega 64 so its quite old but I can still play games like Gotham Knights, Spider-Man in 4k60fps with FSR set to quality. The visual improvement isnt that huge (mostly objects and edges are better in far distance but close-up its really hard to tell) between FSR 2.2.1 and 3.1 but surely a nice to have. And have nice day guys.

**Comment by cdozdil** [comment]
Hi,

Please check if you have `amd_fidelityfx_dx12.dll` and `amd_fidelityfx_vk.dll` next to OptiScaler dll in game folder. If you have the files please set `LogLevel=0` and `LogToFile=true` from `nvngx.ini` and send the `OptiScaler.log` here.

**Comment by WATTASIGMA** [comment]
Hey,

So u were right about the `amd_fidelityfx_dx12.dll` it was missing. I probably didnt see that in the readme. You are a life saver, thank you. I also compared FSR 2.2.1 and 3.1 in Gotham Knights and it was exactly like I said, only far objects/edges are much better but I also had ~5% less fps but in my case the trade-off is acceptable. And thank you again for the great mod and I also wanted to ask if you are planning to also support DLSS1 games. Ofc its not a must, I'm just curios since I really like Final Fantasy XV and there are like 10 games anyway with DLSS1 so its probably not worth it to support these. Back then I used another mod to swap DLSS with FSR in FFXV and I was able to select DLSS ingame but it showed me a blackscreen but the game ran fine and I was still able to run, jump and even see the UI and marks. Anyway, thanks again and have a nice day.
